pkg/server/controllers.go

```diff
@@ -680,6 +632,70 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 	})
 }
 
+func (s *Server) installAPIBinderController(ctx context.Context, config *rest.Config, server *genericapiserver.GenericAPIServer) error {
+	// Client used to create APIBindings within the initializing workspace
+	config = rest.CopyConfig(config)
+	kcpclienthelper.SetMultiClusterRoundTripper(config)
+	config = rest.AddUserAgent(config, initialization.ControllerName)
+	config.Host = fmt.Sprintf("https://%v/services/%v/%v", s.GenericConfig.ExternalAddress, initializingworkspaces.VirtualWorkspaceName, tenancyv1alpha1.ClusterWorkspaceAPIBindingsInitializer)
```

Comment:
would prefer this logic near to the RootPathResolver in the VW, so we know there is a dependency.